extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -41,6 +41,9 @@ class AmpFitText extends AMP.BaseElement {
     /** @private {number} */
     this.maxFontSize_ = -1;
 
+    /** @private {unlistenDef} */
+    this.resizeObserverUnlistener_ = null;
```

Comment:
Should this be initialized to `() => {}` so that we can never accidentally call `null` in the `unlayoutCallback`?

Reply:
it is also used as a signal to tell if resizeObserver has been initialized. I recommend initialize it to `null` but add the check to `unlayoutCallback`

Reply:
Works for me.

Reply:
I set this to null to indicate that a resizeObserver has not been attached yet.

Reply:
Ah! I replied before I saw your comment, SGTM.